packages/react-composites/src/composites/CallComposite/pages/DtmfDialpadPage.tsx

```diff
@@ -70,6 +72,7 @@ const DtmfDialpadPageContent = (props: DialpadPageContentProps): JSX.Element =>
           }}
           longPressTrigger={props.mobileView ? 'touch' : 'mouseAndTouch'}
           dialpadMode={'dtmf'}
+          dtmfAudioContext={props.compositeAudioContext}
```

Comment:
Is this `AudioContext` provided by contoso or it is internally passed?

Reply:
This audio context is provided to the dialpad from the composite. it will be provided to the dialpad from contoso if they are to use just the component in their application.